x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -232,6 +232,8 @@ public class Constants {
         "cluster:admin/xpack/ml/upgrade_mode",
         "cluster:admin/xpack/monitoring/bulk",
         "cluster:admin/xpack/monitoring/migrate/alerts",
+        "cluster:admin/xpack/query_rule/delete",
```

Comment:
since the URL paths for these APIs that manage individual query rules are nested under `/_query_rules/rule` I wonder if we should keep the same pattern here and have `"cluster:admin/xpack/query_rules/rule/delete"` instead.

Reply:
This makes sense, and also helps differentiate them from the ruleset constants. I will update.